code/LeftAndMain.php

```diff
@@ -53,6 +53,7 @@
 use SilverStripe\View\ArrayData;
 use SilverStripe\View\Requirements;
 use SilverStripe\View\SSViewer;
+use stdClass;
```

Comment:
```suggestion
```
Doesn't seem to be used (if it is, please say which line it's on 'cause I missed it)

Reply:
Removed